sdk/eventhub/Azure.Messaging.EventHubs.Processor/samples/Sample01_HelloWorld.md

```diff
@@ -39,7 +39,7 @@ try
             // decision would have to be made as to whether the event should
             // be dropped or published on its own.
 
-            return;
+            break;
```

Comment:
Great catch.... thank you!    This code gets generated from a snippet and this change will be lost.   Do you mind adjusting line 97 [here](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Snippets/Sample01_HelloWorldLiveTests.cs#L97) and line 107 [here](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/tests/Snippets/Sample01_HelloWorldLiveTests.cs#L107)?    I appear to have the same bug.